linode/manager
# Refactor: [M3-6333] - MUI v5 Migration - `Components > EntityDetail`

## Description 📝
Refactor `EntityDetail` to `styled` api away from `jss`.

## Preview 📷
| Screenshot   |
| ------- |
| <img width="1200" alt="Screen Shot 2023-05-16 at 10 58 44 AM" src="https://github.com/linode/manager/assets/125309814/870a00ab-994c-44f1-b771-14d77b3e2056"> |

## How to test 🧪
1. `yarn dev`
2. Go to a linodes detail page: http://localhost:3000/linodes/ {id}
3. Observe there are no visual changes
